### PR TITLE
Add pos_receipt_xml_header_footer

### PR DESCRIPTION
**Pull request by carmenbianca**
Task: https://gestion.coopiteasy.be/web#id=7746&view_type=form&model=project.task

Commit message + readme describes fairly well what's going on here.

### CODE REVIEW

**Comment by carmenbianca** [comment]
As a general observation: Should this be done upstream instead of in a module?

File: pos_receipt_xml_header_footer/readme/USAGE.rst [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+the following:
+
+1. Open the Point of Sale application.
+2. Go to Configuration → Point of Sale, and select your POS.
+3. Enable 'Header & Footer'.
+4. Prepend your header or footer with a single line containing ``<!DOCTYPE QWEB``.
+
+For example, to use line breaks::
+
+  <!DOCTYPE QWEB
```
Comment (huguesdk) [inline pos_receipt_xml_header_footer/readme/USAGE.rst:11]: shouldn’t this tag be closed? is this an odoo quirk?

**Review by huguesdk — COMMENTED** [review]
wow, well done! honestly, you’re way ahead of me there with qweb templates. i trust you on this.

File: pos_receipt_xml_header_footer/readme/USAGE.rst (hunk 1 above)
Comment (carmenbianca) [reply pos_receipt_xml_header_footer/readme/USAGE.rst:11]: Odoo quirk. My editor automatically closing the tag caused me at least 30 minutes of frustration :(

**Review by remytms — APPROVED** [review]
Well done. Even better than overwriting the javascript function ! Thanks a lot.

**Comment by legalsylvain** [comment]
why not propose it to OCA ?

**Comment by carmenbianca** [comment]
Sure, here's the OCA PR: https://github.com/OCA/pos/pull/887

**Pull request closed without merging**